.github/workflows/dotnetcore.yml

```diff
@@ -25,7 +25,10 @@ jobs:
     - name: Build Reason
       run: echo ${{github.ref}} and ${{github.event_name}}
     - name: Build with dotnet
-      run: find . -path ./template -prune -false -o -name '*.csproj' -execdir dotnet build --configuration Release --nologo {} \;
+      run: >-
```

Comment:
So this used to find all csproj except the template csproj and build them separately. I can't remember why I did this though, I think it was to do with the version of Carter the template was pointing at but I'm not entirely sure =)